pkg/cli/cvd.go

```diff
@@ -247,6 +252,38 @@ func (c *cvdCreator) createWithOpts() ([]*hoapi.CVD, error) {
 	return res.CVDs, nil
 }
 
+func (c *cvdCreator) createCVDFromLocalArtifacts() ([]*hoapi.CVD, error) {
+	srcs, err := loadLocalArtifactsSrcs(c.opts.LocalArtifacts)
```

Comment:
instead of list up necessary artifacts in toml file, how about using *-imgs.zip as HO for AB and fetch_cvd does?

Reply:
`ImagesZip` is a valid option as an entry for the local artifacts config file as it's what AB gives you, I'm going to add it in a follow up PR. In such case the config file would have two entries only:

```
ImagesZip="PATH/-imgs.zip"
CVDHostPackageTar="PATH/cvd-host_package.tar.gz"
```